library/ssl_tls13_keys.c

```diff
@@ -180,23 +181,36 @@ int mbedtls_ssl_tls13_hkdf_expand_label(
                                  &hkdf_label_len );
 
     status = psa_key_derivation_setup( &operation, PSA_ALG_HKDF_EXPAND( hash_alg ) );
-    if (status == PSA_SUCCESS)
-        status |= psa_key_derivation_input_bytes( &operation,
-                                                  PSA_KEY_DERIVATION_INPUT_SECRET,
-                                                  secret,
-                                                  secret_len );
-    if (status == PSA_SUCCESS)
-        status |= psa_key_derivation_input_bytes( &operation,
-                                                  PSA_KEY_DERIVATION_INPUT_INFO,
-                                                  hkdf_label,
-                                                  hkdf_label_len );
-    if (status == PSA_SUCCESS)
-        status |= psa_key_derivation_output_bytes( &operation,
-                                                   buf,
-                                                   buf_len );
-    if (status == PSA_SUCCESS)
-        status |= psa_key_derivation_abort( &operation );
 
+    if( status != PSA_SUCCESS )
+         goto cleanup;
+
+    status = psa_key_derivation_input_bytes( &operation,
+                                             PSA_KEY_DERIVATION_INPUT_SECRET,
+                                             secret,
+                                             secret_len );
+
+    if( status != PSA_SUCCESS )
+         goto cleanup;
+
+    status = psa_key_derivation_input_bytes( &operation,
+                                             PSA_KEY_DERIVATION_INPUT_INFO,
+                                             hkdf_label,
+                                             hkdf_label_len );
+
+    if( status != PSA_SUCCESS )
+         goto cleanup;
+
+    status = psa_key_derivation_output_bytes( &operation,
+                                              buf,
+                                              buf_len );
+
+    if( status != PSA_SUCCESS )
```

Comment:
This code is a no-op, but that's not worth triggering the CI again :)

Reply:
Yes, copy paste error. Compiler should handle this.
On the other hand it might be needed if extra code is added, but probably this won't happen.